core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.scss

```diff
@@ -0,0 +1,14 @@
+:host {
+    ::ng-deep {
+        .p-dialog {
+            .p-dialog-header {
+                padding: 1.5rem 2.5rem;
+            }
+
+            .p-dialog-content {
+                padding: 0;
+                overflow: hidden;
+            }
+        }
```

Comment:
Should we move this to global?

Reply:
No, also we don't need to make it look like our current edit dialog, we can leave it as it is